omnigan/trainer.py

```diff
@@ -214,9 +214,7 @@ def resume_from_path(
         if new_exp:
             exp = Experiment(project_name="omnigan", **comet_kwargs)
             exp.log_asset_folder(
-                str(Path(__file__).parent),
-                recursive=True,
-                log_file_name=True,
+                str(Path(__file__).parent), recursive=True, log_file_name=True,
```

Comment:
what's going on here? Thant's what I have on master
![2020-11-04 at 19 19](https://user-images.githubusercontent.com/9283470/98182016-b7c46900-1ed2-11eb-8a43-bc55d312fd76.png)

Reply:
Why does it say masster has log_asset_folder on multiple lines?

Reply:
weird indeed.. not sure why it did that

Reply:
Thought it was just black that reformatted the line..

Reply:
Yeah, anyway, doesn't really matter since the right version is in master